app/controllers/application_controller/buttons.rb

```diff
@@ -116,7 +116,7 @@ def automate_button_field_changed
         page.replace("ab_form", :partial => "shared/buttons/ab_form")
       end
       if params[:visibility_typ]
-        page.replace("form_role_visibility", :partial => "layouts/role_visibility", :locals => {:rec_id => "#{@custom_button.id || "new"}", :action => "automate_button_field_changed"})
+        page.replace("form_role_visibility", :partial => "layouts/role_visibility", :locals => {:rec_id => (@custom_button.id || "new").to_s, :action => "automate_button_field_changed"})
```

Comment:
Yes, I can do this one.  Note, there are many like this so I didn't change them to minimize the logical changes.